### ISSUE TITLE
The shared link could not be updated

### ISSUE THREAD

**Issue by AbnerMai** [minio/console#1007]
Operation: I select a time greater than 7 days by specifying a date
Result: the page reports an error and the link is empty

![image](https://user-images.githubusercontent.com/26241803/131987004-c64145e8-db6d-4183-9d3f-3b662e32340f.png)

Operation: I reselect the date less than 7 days. The page is normal, but the link is empty

![image](https://user-images.githubusercontent.com/26241803/131986955-9abb4a50-af29-4c48-8370-f54fe60f0b25.png)

**Comment by harshavardhana** [comment]
> Operation: I select a time greater than 7 days by specifying a date

AWS S3 API does not allow presigned shared link beyond 7 days - this is working as expected.

**Comment by AbnerMai** [comment]
Yes, but I mean the time has been changed and the link has not been updated

**Comment by bexsoft** [comment]
Couldn't reproduce the issue, but will test further

**Comment by bexsoft** [comment]
Issue couldn't be reproduced with latest version, 
Would you please tell us which version of console, browser & MinIO are you using?

![2021-09-09 12-04-03 2021-09-09 12_06_53](https://user-images.githubusercontent.com/33497058/132731222-6a720725-037b-4c4c-a2e1-34c41dec125e.gif)


Thank you!

**Comment by harshavardhana** [comment]
> Would you please tell us which version of console, browser & MinIO are you using?

adding a year here is not meaningful @bexsoft since pre-signed can never be more than 7 days. I believe that should be fixed.

**Comment by bexsoft** [comment]
Sure

> > Would you please tell us which version of console, browser & MinIO are you using?
> 
> adding a year here is not meaningful @bexsoft since pre-signed can never be more than 7 days. I believe that should be fixed.

Sure @harshavardhana, we will change this date selector

**Comment by AbnerMai** [comment]
Sorry, it may be the cause of my browser's cache. After I emptied the cache, I did not find the problem, but the date picker can select the previous date, and there is no error. The link is still the previous link when it was correct. If the year selector is changed, the time is before today tomorrow, and no error is reported.
![minio1](https://user-images.githubusercontent.com/26241803/132788874-0216fc87-a393-447e-8866-c8d6b0dd3515.gif)

MinIO Server: RELEASE.2021-09-03T03-56-13Z
MinIO Console:  v0.9.6
